app/main.py

```diff
@@ -16,4 +16,17 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [
+        my_favourite_films,
+        marks,
+        collection_of_coins
+    ],
+    "immutable": [
+        lucky_number,
+        pi,
+        one_is_a_prime_number,
+        name,
+        profile_info
+    ]
+}
```

Comment:
The `collection_of_coins` is categorized under 'mutable', but sets in Python are mutable. Consider moving `collection_of_coins` to the 'immutable' category if you intend to keep only immutable types there.